.devcontainer/install-dependencies.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cd server && POETRY_VIRTUALENVS_CREATE=false poetry install # give editor intellisense
```

Comment:
Add docker compose build...

So when they run docker compose up, it's faster

Reply:
Put the cd server and the other command in the parentheses, so it doesn't affect your current working directory